src/main/resources/Diagram/WebHome.xml

```diff
@@ -64,14 +68,14 @@
 {{/velocity}}
 
 {{velocity}}
-#set ($diagramClassReference = $services.model.createDocumentReference('', 'Diagram', 'DiagramClass'))
-#if ($services.licensing.licensor.hasLicensureForEntity($diagramClassReference))
+#if ($services.licensing.licensor.hasLicensureForEntity($services.model.createDocumentReference('', 'Diagram', 'DiagramClass')))
```

Comment:
```suggestion
#if ($services.licensing.licensor.hasLicensureForEntity(
    $services.model.createDocumentReference('', 'Diagram', 'DiagramClass')))
```

The best practice is to wrap long lines (>120 chars) in order to avoid horizontal scroll bars when viewing the source code.